docs/guides/digital_ocean.md

```diff
@@ -0,0 +1,37 @@
+# Deploying with digital ocean
+## single machine with docker
+### deploying
+Prerequisite:
+- make sure you have completed the docker deployment guide
+- you know how to create a 'one-click app' droplet
+- you know how to ssh in a digital ocean droplet
+- you have published your image to dockerhub
+
+- Start by creating a new droplet. Click on the 'one-click apps' and choose docker. It will create a droplet for you with docker already installed
+
+- ssh into your droplet and start by upgrading it `apt-get upgrade`. It might be missing some security upgrades
+
+- create a directory for your configuration files for example `mkdir /etc/opt/app `
+(etc is the directory mainly used for storing config files on linux, opt is generally used for things not necessary to the OS). Then cd into that directory
+
+- copy over your `.env` files and your docker-compose.yml file
+on your droplet `mkdir config`
+on your computer
+`scp ./config/docker.env root@my_droplet_ip:/etc/opt/app/config`
+`scp docker-compose.yml root@my_droplet_ip:/etc/opt/app`
+
+- login with docker (needed in order to pull the image from dockerhub)
+`docker login` (then follow the instructions)
+
+- start a swarm on your droplet (don't forget to replace my_droplet_ip_address with your droplet's ip address)
+`docker swarm init --advertise-addr my_droplet_ip_address --listen-addr my_droplet_ip_address`
+
+- then deploy your stack with
+`docker stack deploy -c docker-compose.yml my_app_name`
+
+### upgrading
```

Comment:
I am still not 100% sure that this is the recommended way to proceed. I am testing stuff with `docker stack deploy` here as well. Any advice here welcome